google/cloud/storagecontrol/v2/samples/storage_control_folder_samples.cc

```diff
@@ -147,11 +147,11 @@ void AutoRun(std::vector<std::string> const& argv) {
   auto client = storagecontrol::StorageControlClient(
       storagecontrol::MakeStorageControlConnection());
   auto generator = google::cloud::internal::DefaultPRNG(std::random_device{}());
-  auto const prefix = "storage-control-samples";
-  auto const folder_id = prefix + std::string{"-"} +
+  auto const prefix = std::string{"storage-control-samples"};
```

Comment:
Good idea, but unrelated to this PR or did `clang-tidy` say something?

Reply:
It wanted `auto const* const`